### PR TITLE
added support for several european locales

### PR DESCRIPTION
**Pull request by bancer**
fixes cakephp/cakephp#8712

### CODE REVIEW

File: lib/Cake/I18n/L10n.php [hunk 1]
```diff
@@ -304,6 +312,7 @@ class L10n {
 		'sq' => array('language' => 'Albanian', 'locale' => 'sqi', 'localeFallback' => 'sqi', 'charset' => 'utf-8', 'direction' => 'ltr'),
 		'sr' => array('language' => 'Serbian', 'locale' => 'srp', 'localeFallback' => 'srp', 'charset' => 'utf-8', 'direction' => 'ltr'),
 		'sv' => array('language' => 'Swedish', 'locale' => 'swe', 'localeFallback' => 'swe', 'charset' => 'utf-8', 'direction' => 'ltr'),
+		'sv-se' => array('language' => 'Swedish (Sweden)', 'locale' => 'sv_se', 'localeFallback' => 'swe', 'charset' => 'utf-8', 'direction' => 'ltr'),
```
Comment (dereuromark) [inline lib/Cake/I18n/L10n.php]: They are usually in alphabetical order.

File: lib/Cake/I18n/L10n.php [hunk 2]
```diff
@@ -286,16 +290,20 @@ class L10n {
 		'ms' => array('language' => 'Malaysian', 'locale' => 'msa', 'localeFallback' => 'msa', 'charset' => 'utf-8', 'direction' => 'ltr'),
 		'mt' => array('language' => 'Maltese', 'locale' => 'mlt', 'localeFallback' => 'mlt', 'charset' => 'utf-8', 'direction' => 'ltr'),
 		'nb' => array('language' => 'Norwegian Bokmal', 'locale' => 'nob', 'localeFallback' => 'nor', 'charset' => 'utf-8', 'direction' => 'ltr'),
+		'nb-no' => array('language' => 'Norwegian Bokmål (Norway)', 'locale' => 'nb_no', 'localeFallback' => 'nor', 'charset' => 'utf-8', 'direction' => 'ltr'),
 		'nl' => array('language' => 'Dutch (Standard)', 'locale' => 'nld', 'localeFallback' => 'nld', 'charset' => 'utf-8', 'direction' => 'ltr'),
+		'nl-nl' => array('language' => 'Dutch (Netherlands)', 'locale' => 'nl_nl', 'localeFallback' => 'nld', 'charset' => 'utf-8', 'direction' => 'ltr'),
 		'nl-be' => array('language' => 'Dutch (Belgium)', 'locale' => 'nl_be', 'localeFallback' => 'nld', 'charset' => 'utf-8', 'direction' => 'ltr'),
```
Comment (dereuromark) [inline lib/Cake/I18n/L10n.php]: nitpick: nl-be would come before nl-nl IMO

**Comment by dereuromark** [comment]
Looks like this also needs porting to 3.x.

**Comment by markstory** [comment]
Why? There is no L10n class in 3.x

**Comment by dereuromark** [comment]
Ah, right, gotcha.

**Pull request merged**